client/components/data/viewers-data/index.jsx

```diff
@@ -3,9 +3,8 @@
  *
```

Comment:
👍  Smoke tested and learned about viewers (only available on private, non-Jetpack sites).